release/sdk-config.yaml.tmpl

```diff
@@ -34,3 +34,6 @@ commands:
 - name: ide
   path: damlc/da-hs-damlc-app
   args: ["lax", "ide"]
+- name: test
+  path: daml-helper/daml-helper
+  args: ["test"]
```

Comment:
The `test` command needs a `desc:` description otherwise it will be hidden in `daml -h`. 

Like Moritz said, I think it makes more sense to have this call `damlc test` directly and have that command be aware of projects, like `daml build` already is. I searched the docs and everything and I couldn't find any instance of `damlc test` being invoked in official docs (outside of `daml-foundations/daml-ghc/README.md`), so I think it's safe to change the `test` command.

Ideally the two usage modes for `damlc test` would be:

```
damlc test           Tests the current project
damlc test FILE      Tests the specific daml file.
```

The former would test the current project (and error out if run outside of a project), and the latter would test a file and its dependencies, regardless of being in a project, to preserve compatibility with existing workflows.